### DISCUSSION TITLE
How to specify partially merged paired-end library

### DISCUSSION THREAD

**Discussion by sivico26** [ablab/spades#751] [Q&A] [answered]
Hi, I'm trying to use Spades to assemble a small (~250 Mb) tetraploid plant genome. I have 2x150 Illumina paired-end data and a few Nanopore reads.

As part of preprocessing for Illumina data, I did error correction with [BFC](https://github.com/lh3/bfc), and given the high variance of the insert size, merging with BBMerge, which result in  ~50% of the reads merged.

I tried using Spades specifying the 3 files: `-1 unmerged_1.fastq -2 unmerged_2.fastq --merged merged.fastq --nanopore ont_reads.fastq`

But got quite poor results:
```
$ abyss-fac scaffolds.fasta 
n	n:500	L50	min	N75	N50	N25	E-size	max	sum	name
188699	800	45	500	1926	6212	86398	75217	356847	2255762	scaffolds.fasta
```

[spades.log](https://github.com/ablab/spades/files/3272021/spades.log)
[warnings.log](https://github.com/ablab/spades/files/3272022/warnings.log)

So I wonder if in this case this is the best parameter configuration I can provide to Spades, or if there is a better way to do it. Also, do you think it would be better to provide the reads before the merging? 

Thanks in advance

**Comment by asl** [comment, marked as answer]
You're providing the reads improperly:

1. For your merged reads you need to provide also the original unmerged reads
2. The "unmerged" part should be provided as a separate paired-end library

Why do you think that the results are poor given the complexity of the genome?
